src/app/shell/personal-cabinet/parent/create-application/create-application.component.ts

```diff
@@ -53,6 +53,7 @@ export class CreateApplicationComponent implements OnInit, OnDestroy {
   destroy$: Subject<boolean> = new Subject<boolean>();
 
   ChildFormControl = new FormControl('', Validators.required);
+  isAllowed: boolean = true;
```

Comment:
You have two values for the same thing
Use isAllowedNewApplicatiom

Reply:
it seems I cannot use 'isAllowedNewApplication' for mat-select and small tag because the state is false by default. It means a red message will display by default. Maybe 'isAllowed' will be ok for the all cases instead of 'isAllowedNewApplicatiom'

Reply:
Then make state true by default:)